1-js/02-first-steps/06-alert-prompt-confirm/article.md

```diff
@@ -1,105 +1,108 @@
 
-It shows a modal window with a text message, an input field for the visitor, and the buttons OK/Cancel.
+Mostra uma janela modal com uma mensagem de texto, um campo de entrada para o visitante, e os botões OK/CANCEL
 
 `title`
-: The text to show the visitor.
-
+: Texto para ser mostrado ao visitante.
 `default`
-: An optional second parameter, the initial value for the input field.
+: Um parâmetro opcional, o valor inicial para o campo de entrada.
 
-```smart header="The square brackets in syntax `[...]`"
-The square brackets around `default` in the syntax above denote that the parameter as optional, not required.
+```smart header="Os colchetes na sintaxe `[...]`"
+Os colchetes ao redor de `default` na sintaxe acima denota que o parâmetro como opcional, não é obrigatório.
 ```
 
-The visitor can type something in the prompt input field and press OK. Then we get that text in the `result`. Or they can cancel the input by pressing Cancel or hitting the `key:Esc` key, then we get `null` as the `result`.
+O visitante pode digitar algo no campo de entrada do prompt e pressionar OK. Então nós temos esse texto no `result`  Ou ele pode cancelar a entrada pressionando Cancelar ou `key:Esc`, então nos temos `null` como o `result`.
 
-The call to `prompt` returns the text from the input field or `null` if the input was canceled.
+A chamada do `prompt` retorna o texto do campo de entrada ou `null` se a entrada for cancelada.
 
-For instance:
+Por exemplo:
 
 ```js run
-let age = prompt('How old are you?', 100);
+let age = prompt('Qual a sua idade ?', 100);
 
-alert(`You are ${age} years old!`); // You are 100 years old!
+alert(`Você tem ${age} anos!`); //Você tem 100 anos!
 ```
 
-````warn header="In IE: always supply a `default`"
-The second parameter is optional, but if we don't supply it, Internet Explorer will insert the text `"undefined"` into the prompt.
+``warn header="No IE: sempre forneça um `default`"
+O segundo parâmetro é opcional, mas se não o fornecermos, o Internet Explorer irá inserir o texto `"undefined"` no prompt.
 
-Run this code in Internet Explorer to see:
+Execute este código no Internet Explorer para visualizar:
 
 ```js run
-let test = prompt("Test");
+let test = prompt('Teste');
 ```
 
-So, for prompts to look good in IE, we recommend always providing the second argument:
+Portanto, para que os prompts tenham boa aparência no IE, recomendamos que sempre forneça o segundo argumento:
 
 ```js run
-let test = prompt("Test", ''); // <-- for IE
+let test = prompt('Teste', ''); // <-- para o IE
 ```
+
 ````
 
 ## confirm
 
-The syntax:
+Sintaxe:
 
 ```js
 result = confirm(question);
 ```
 
-The function `confirm` shows a modal window with a `question` and two buttons: OK and Cancel.
-
-The result is `true` if OK is pressed and `false` otherwise.
-
-For example:
+A função `confirm` mostra uma janela modal com uma `question` e dois botões: OK e Cancelar.
+O resultado é `true` se OK for pressionado e `false` caso contrário
+Por exemplo:
```

Comment:
keep a new empty line in between those lines as in the original it makes it easier to merge in the future.